app/pb_kits/playbook/pb_radio/_radio.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= content_tag(:label,
+    id: object.id,
+    data: object.data,
+    text: object.text,
+    value: object.value,
+    checked: object.checked,
+    class: object.classname) do %>
+
+    <%= radio_button_tag object.name, object.value, object.selected %>
+    <span class="pb_radio_button">
+      <div class="radio">
+        <% if object.children %>
+        <%= capture(&object.children) %>
+      <% else %>
+        <%= object.input %>
+      <% end %>
+      </div>
+    </span>
+    <%= pb_rails("body", props: { text: object.text, dark: object.dark }) %>
```

Comment:
This does not handle a block correctly. The children, in this case, we should expect it to be a radio input. In yours you are adding that button always. You should add a specific example for the docs ad update the logic like I outlined. Use select kit, or textarea kit as examples